elements/lisk-chain/src/transactions/transactions_handlers.ts

```diff
@@ -23,32 +23,14 @@ import { DataAccess } from '../data_access';
 import { StateStore } from '../state_store';
 import {
 	Contexter,
-	ExceptionOptions,
 	MatcherTransaction,
 	WriteableTransactionResponse,
 } from '../types';
 
-import * as exceptionsHandlers from './exceptions_handlers';
-import * as votesWeight from './votes_weight';
-
-export const validateTransactions = (exceptions?: ExceptionOptions) => (
+export const validateTransactions = () => (
 	transactions: ReadonlyArray<BaseTransaction>,
-): ReadonlyArray<TransactionResponse> => {
```

Comment:
since we no longer have exceptions.. why don't we change function to??
```
export const validateTransactions = (transactions: ReadonlyArray<BaseTransaction>): ReadonlyArray<TransactionResponse> => 
```
```

Reply:
I think this was originally to make all the transaction interface the same.
Also, composeTransactions relies on this

Reply:
and the `composeTransactions` was designed to address this exceptions if i understand correctly? I don't have the context though. but if it doesn't make sense now, its better to clean it up now. otherwise you can ignore this comment 👍

Reply:
`composeTransactions` was just a way to handle multiple cases for transaction validation and keeping the `context` while injecting to the transaction pool.
Ill create an issue to address this

Reply:
Created: https://github.com/LiskHQ/lisk-sdk/issues/5088